packages/dashboard/src/components/appbar.tsx

```diff
@@ -180,6 +181,16 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
     }
   }
 
+  React.useEffect(() => {
```

Comment:
We should add a try here to handle a possible error.

Reply:
Yup, makes sense, added https://github.com/open-rmf/rmf-web/pull/702/commits/294ff1214347be2240558574622e4e8395a33d7e

Reply:
Thanks!